docs/maui/alerts/snackbar.md

```diff
@@ -116,4 +116,104 @@ You can find the source code for `Snackbar` over on the [.NET MAUI Community Too
 
 1. The API allows override existing methods with your own implementation or even create your own Snackbar, by implementing `ISnackbar` interface.
 2. "Native" Snackbar is available only on Android and created by Google. Other platforms use "Container" (`UIView` for iOS and MacCatalyst, `ToastNotification` on Windows).
-3. `Snackbar` on Windows can't be anchored to `VisualElement` and is always displayed as a default Windows Notification.
\ No newline at end of file
+3. `Snackbar` on Windows can't be anchored to `VisualElement` and is always displayed as a default Windows Notification.
+
+### WinUI specifics
+
+`ToastNotification` which is used to show `Snackbar` on Windows has 2 types of activation: foreground and background.
+
+Foreground activation type is used in `CommunityToolkit.Maui` library. That means the new instance of application is executed. It is up to the developer how to handle such situation. There are some proposals:
```

Comment:
```suggestion
Foreground activation type is used in `CommunityToolkit.Maui` library. That means, whenever a notification is shown a new instance of application is executed. It is up to the developer how to handle such situations. Here are a few suggestions:
```

I don't know anything about this, so please check if what I corrected makes any sense :D also do we have a link to some documentation on the foreground and background activation stuff?